projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts

```diff
@@ -123,6 +123,10 @@ export class DisplayEditComponent implements OnInit {
         this.readOnlyValue = this._valueTypeService.isReadOnly(this.valueTypeOrClass);
     }
 
+    getTooltipText(): string {
+        return 'Value creation date: ' + this.displayValue.valueCreationDate + '\n Attached to user: ' + this.displayValue.attachedToUser;
```

Comment:
does a newline take any effect in html?

Reply:
yeah, it does

Reply:
but isn't it treated like a newline (rather than an actual line break)?

Reply:
ah yeah, it doesn't generate a `<br>`. Here's the generated HTML:
<img width="1150" alt="Screen Shot 2020-09-17 at 11 03 57" src="https://user-images.githubusercontent.com/60604010/93449945-a64ce080-f8d5-11ea-8d5f-c26a23fec7dd.png">